### ISSUE TITLE
build failed

### ISSUE THREAD

**Issue by pennpeng** [rh-openjdk/redhat-openjdk-containers#44]
```
# cekit build
2018-11-27 10:51:52,989 cekit        INFO     Generating files for docker engine.
2018-11-27 10:51:53,057 cekit        INFO     Initializing image descriptor...
2018-11-27 10:51:59,832 cekit        WARNING  The way of defining repository 'jboss-os' is deprecated. Convert it to an URL based repository object. Consult Cekit docs, for more details.
2018-11-27 10:51:59,833 cekit        ERROR    Package repository 'jboss-os' used in descriptor is not available in Cekit configuration file. 

```

**Comment by pennpeng** [comment]
```
# cekit build -v
2018-11-27 11:36:51,458 cekit        DEBUG    Running version 2.1.4
2018-11-27 11:36:51,460 cekit        DEBUG    Removing dirty directory: 'target/repo'
2018-11-27 11:36:51,543 cekit        INFO     Generating files for docker engine.
2018-11-27 11:36:51,543 cekit        DEBUG    Loading descriptor from path 'image.yaml'.
2018-11-27 11:36:51,580 cekit        INFO     Initializing image descriptor...
2018-11-27 11:36:51,580 cekit        DEBUG    Retrieving module repositories for 'centos/openjdk-8-centos7'
2018-11-27 11:36:51,581 cekit        DEBUG    Preparing resource 'cct_module'
2018-11-27 11:36:51,582 cekit        DEBUG    Running 'git clone --depth 1 https://github.com/jboss-openshift/cct_module.git target/repo/cct_module.git-master -b master'
2018-11-27 11:37:23,523 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/datagrid-openshift/module.yaml'.
2018-11-27 11:37:23,537 cekit        DEBUG    Retrieving module repositories for 'datagrid-openshift'
2018-11-27 11:37:23,537 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,537 cekit        DEBUG    Adding module 'datagrid-openshift', path: 'target/repo/cct_module.git-master/datagrid-openshift'
2018-11-27 11:37:23,538 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/datagrid7-openshift/module.yaml'.
2018-11-27 11:37:23,545 cekit        DEBUG    Retrieving module repositories for 'datagrid7-openshift'
2018-11-27 11:37:23,545 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,545 cekit        DEBUG    Adding module 'datagrid7-openshift', path: 'target/repo/cct_module.git-master/datagrid7-openshift'
2018-11-27 11:37:23,546 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/dynamic-resources/module.yaml'.
2018-11-27 11:37:23,555 cekit        DEBUG    Retrieving module repositories for 'dynamic-resources'
2018-11-27 11:37:23,556 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,556 cekit        DEBUG    Adding module 'dynamic-resources', path: 'target/repo/cct_module.git-master/dynamic-resources'
2018-11-27 11:37:23,556 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/java-alternatives/module.yaml'.
2018-11-27 11:37:23,561 cekit        DEBUG    Retrieving module repositories for 'java-alternatives'
2018-11-27 11:37:23,562 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,562 cekit        DEBUG    Adding module 'java-alternatives', path: 'target/repo/cct_module.git-master/java-alternatives'
2018-11-27 11:37:23,562 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss-maven/module.yaml'.
2018-11-27 11:37:23,566 cekit        DEBUG    Retrieving module repositories for 'jboss-maven'
2018-11-27 11:37:23,567 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,567 cekit        DEBUG    Adding module 'jboss-maven', path: 'target/repo/cct_module.git-master/jboss-maven'
2018-11-27 11:37:23,567 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/amq/s2i/bash/module.yaml'.
2018-11-27 11:37:23,575 cekit        DEBUG    Retrieving module repositories for 'jboss.container.amq.s2i.bash'
2018-11-27 11:37:23,576 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,576 cekit        DEBUG    Adding module 'jboss.container.amq.s2i.bash', path: 'target/repo/cct_module.git-master/jboss/container/amq/s2i/bash'
2018-11-27 11:37:23,577 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/eap/s2i/bash/module.yaml'.
2018-11-27 11:37:23,587 cekit        DEBUG    Retrieving module repositories for 'jboss.container.eap.s2i.bash'
2018-11-27 11:37:23,587 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,588 cekit        DEBUG    Adding module 'jboss.container.eap.s2i.bash', path: 'target/repo/cct_module.git-master/jboss/container/eap/s2i/bash'
2018-11-27 11:37:23,588 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/hawkular/api/module.yaml'.
2018-11-27 11:37:23,618 cekit        DEBUG    Retrieving module repositories for 'jboss.container.hawkular.api'
2018-11-27 11:37:23,618 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,618 cekit        DEBUG    Adding module 'jboss.container.hawkular.api', path: 'target/repo/cct_module.git-master/jboss/container/hawkular/api'
2018-11-27 11:37:23,619 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/hawkular/bash/module.yaml'.
2018-11-27 11:37:23,638 cekit        DEBUG    Retrieving module repositories for 'jboss.container.hawkular.bash'
2018-11-27 11:37:23,639 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,639 cekit        DEBUG    Adding module 'jboss.container.hawkular.bash', path: 'target/repo/cct_module.git-master/jboss/container/hawkular/bash'
2018-11-27 11:37:23,640 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/java/jvm/api/module.yaml'.
2018-11-27 11:37:23,666 cekit        DEBUG    Retrieving module repositories for 'jboss.container.java.jvm.api'
2018-11-27 11:37:23,667 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,667 cekit        DEBUG    Adding module 'jboss.container.java.jvm.api', path: 'target/repo/cct_module.git-master/jboss/container/java/jvm/api'
2018-11-27 11:37:23,667 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/java/jvm/bash/module.yaml'.
2018-11-27 11:37:23,675 cekit        DEBUG    Retrieving module repositories for 'jboss.container.java.jvm.bash'
2018-11-27 11:37:23,675 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,675 cekit        DEBUG    Adding module 'jboss.container.java.jvm.bash', path: 'target/repo/cct_module.git-master/jboss/container/java/jvm/bash'
2018-11-27 11:37:23,676 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/java/proxy/bash/module.yaml'.
2018-11-27 11:37:23,683 cekit        DEBUG    Retrieving module repositories for 'jboss.container.java.proxy.bash'
2018-11-27 11:37:23,683 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,684 cekit        DEBUG    Adding module 'jboss.container.java.proxy.bash', path: 'target/repo/cct_module.git-master/jboss/container/java/proxy/bash'
2018-11-27 11:37:23,684 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/java/run/bash/module.yaml'.
2018-11-27 11:37:23,700 cekit        DEBUG    Retrieving module repositories for 'jboss.container.java.run.bash'
2018-11-27 11:37:23,701 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,701 cekit        DEBUG    Adding module 'jboss.container.java.run.bash', path: 'target/repo/cct_module.git-master/jboss/container/java/run/bash'
2018-11-27 11:37:23,701 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/java/s2i/bash/module.yaml'.
2018-11-27 11:37:23,711 cekit        DEBUG    Retrieving module repositories for 'jboss.container.java.s2i.bash'
2018-11-27 11:37:23,712 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,712 cekit        DEBUG    Adding module 'jboss.container.java.s2i.bash', path: 'target/repo/cct_module.git-master/jboss/container/java/s2i/bash'
2018-11-27 11:37:23,713 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/jolokia/api/module.yaml'.
2018-11-27 11:37:23,734 cekit        DEBUG    Retrieving module repositories for 'jboss.container.jolokia.api'
2018-11-27 11:37:23,735 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,735 cekit        DEBUG    Adding module 'jboss.container.jolokia.api', path: 'target/repo/cct_module.git-master/jboss/container/jolokia/api'
2018-11-27 11:37:23,736 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/jolokia/bash/module.yaml'.
2018-11-27 11:37:23,765 cekit        DEBUG    Retrieving module repositories for 'jboss.container.jolokia.bash'
2018-11-27 11:37:23,765 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,766 cekit        DEBUG    Adding module 'jboss.container.jolokia.bash', path: 'target/repo/cct_module.git-master/jboss/container/jolokia/bash'
2018-11-27 11:37:23,767 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/jws/s2i/bash/module.yaml'.
2018-11-27 11:37:23,775 cekit        DEBUG    Retrieving module repositories for 'jboss.container.jws.s2i.bash'
2018-11-27 11:37:23,775 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,775 cekit        DEBUG    Adding module 'jboss.container.jws.s2i.bash', path: 'target/repo/cct_module.git-master/jboss/container/jws/s2i/bash'
2018-11-27 11:37:23,776 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/maven/35/bash/module.yaml'.
2018-11-27 11:37:23,784 cekit        DEBUG    Retrieving module repositories for 'jboss.container.maven.35.bash'
2018-11-27 11:37:23,785 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,785 cekit        DEBUG    Adding module 'jboss.container.maven.35.bash', path: 'target/repo/cct_module.git-master/jboss/container/maven/35/bash'
2018-11-27 11:37:23,786 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/maven/api/module.yaml'.
2018-11-27 11:37:23,823 cekit        DEBUG    Retrieving module repositories for 'jboss.container.maven.api'
2018-11-27 11:37:23,824 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,824 cekit        DEBUG    Adding module 'jboss.container.maven.api', path: 'target/repo/cct_module.git-master/jboss/container/maven/api'
2018-11-27 11:37:23,825 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/maven/default/bash/module.yaml'.
2018-11-27 11:37:23,834 cekit        DEBUG    Retrieving module repositories for 'jboss.container.maven.default.bash'
2018-11-27 11:37:23,834 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,835 cekit        DEBUG    Adding module 'jboss.container.maven.default.bash', path: 'target/repo/cct_module.git-master/jboss/container/maven/default/bash'
2018-11-27 11:37:23,835 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/maven/s2i/api/module.yaml'.
2018-11-27 11:37:23,844 cekit        DEBUG    Retrieving module repositories for 'jboss.container.maven.s2i.api'
2018-11-27 11:37:23,844 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,844 cekit        DEBUG    Adding module 'jboss.container.maven.s2i.api', path: 'target/repo/cct_module.git-master/jboss/container/maven/s2i/api'
2018-11-27 11:37:23,845 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/maven/s2i/bash/module.yaml'.
2018-11-27 11:37:23,853 cekit        DEBUG    Retrieving module repositories for 'jboss.container.maven.s2i.bash'
2018-11-27 11:37:23,853 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,854 cekit        DEBUG    Adding module 'jboss.container.maven.s2i.bash', path: 'target/repo/cct_module.git-master/jboss/container/maven/s2i/bash'
2018-11-27 11:37:23,854 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/openjdk/base/module.yaml'.
2018-11-27 11:37:23,860 cekit        DEBUG    Retrieving module repositories for 'jboss.container.openjdk.base'
2018-11-27 11:37:23,860 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,861 cekit        DEBUG    Adding module 'jboss.container.openjdk.base', path: 'target/repo/cct_module.git-master/jboss/container/openjdk/base'
2018-11-27 11:37:23,861 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/openjdk/jdk/10/module.yaml'.
2018-11-27 11:37:23,872 cekit        DEBUG    Retrieving module repositories for 'jboss.container.openjdk.jdk'
2018-11-27 11:37:23,873 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,873 cekit        DEBUG    Adding module 'jboss.container.openjdk.jdk', path: 'target/repo/cct_module.git-master/jboss/container/openjdk/jdk/10'
2018-11-27 11:37:23,874 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/openjdk/jdk/11/module.yaml'.
2018-11-27 11:37:23,887 cekit        DEBUG    Retrieving module repositories for 'jboss.container.openjdk.jdk'
2018-11-27 11:37:23,888 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,888 cekit        DEBUG    Adding module 'jboss.container.openjdk.jdk', path: 'target/repo/cct_module.git-master/jboss/container/openjdk/jdk/11'
2018-11-27 11:37:23,889 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/openjdk/jdk/8/module.yaml'.
2018-11-27 11:37:23,900 cekit        DEBUG    Retrieving module repositories for 'jboss.container.openjdk.jdk'
2018-11-27 11:37:23,901 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,901 cekit        DEBUG    Adding module 'jboss.container.openjdk.jdk', path: 'target/repo/cct_module.git-master/jboss/container/openjdk/jdk/8'
2018-11-27 11:37:23,902 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/proxy/api/module.yaml'.
2018-11-27 11:37:23,914 cekit        DEBUG    Retrieving module repositories for 'jboss.container.proxy.api'
2018-11-27 11:37:23,915 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,915 cekit        DEBUG    Adding module 'jboss.container.proxy.api', path: 'target/repo/cct_module.git-master/jboss/container/proxy/api'
2018-11-27 11:37:23,915 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/s2i/core/api/module.yaml'.
2018-11-27 11:37:23,943 cekit        DEBUG    Retrieving module repositories for 'jboss.container.s2i.core.api'
2018-11-27 11:37:23,944 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,944 cekit        DEBUG    Adding module 'jboss.container.s2i.core.api', path: 'target/repo/cct_module.git-master/jboss/container/s2i/core/api'
2018-11-27 11:37:23,944 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/s2i/core/bash/module.yaml'.
2018-11-27 11:37:23,951 cekit        DEBUG    Retrieving module repositories for 'jboss.container.s2i.core.bash'
2018-11-27 11:37:23,951 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,951 cekit        DEBUG    Adding module 'jboss.container.s2i.core.bash', path: 'target/repo/cct_module.git-master/jboss/container/s2i/core/bash'
2018-11-27 11:37:23,952 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/user/module.yaml'.
2018-11-27 11:37:23,961 cekit        DEBUG    Retrieving module repositories for 'jboss.container.user'
2018-11-27 11:37:23,961 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,961 cekit        DEBUG    Adding module 'jboss.container.user', path: 'target/repo/cct_module.git-master/jboss/container/user'
2018-11-27 11:37:23,962 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jboss/container/util/logging/bash/module.yaml'.
2018-11-27 11:37:23,968 cekit        DEBUG    Retrieving module repositories for 'jboss.container.util.logging.bash'
2018-11-27 11:37:23,969 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,969 cekit        DEBUG    Adding module 'jboss.container.util.logging.bash', path: 'target/repo/cct_module.git-master/jboss/container/util/logging/bash'
2018-11-27 11:37:23,969 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/jolokia/module.yaml'.
2018-11-27 11:37:23,974 cekit        DEBUG    Retrieving module repositories for 'jolokia'
2018-11-27 11:37:23,974 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,974 cekit        DEBUG    Adding module 'jolokia', path: 'target/repo/cct_module.git-master/jolokia'
2018-11-27 11:37:23,975 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/keycloak-layer/module.yaml'.
2018-11-27 11:37:23,978 cekit        DEBUG    Retrieving module repositories for 'keycloak-layer'
2018-11-27 11:37:23,978 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,979 cekit        DEBUG    Adding module 'keycloak-layer', path: 'target/repo/cct_module.git-master/keycloak-layer'
2018-11-27 11:37:23,979 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/openshift-layer/module.yaml'.
2018-11-27 11:37:23,983 cekit        DEBUG    Retrieving module repositories for 'openshift-layer'
2018-11-27 11:37:23,984 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,984 cekit        DEBUG    Adding module 'openshift-layer', path: 'target/repo/cct_module.git-master/openshift-layer'
2018-11-27 11:37:23,984 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/openshift-passwd/module.yaml'.
2018-11-27 11:37:23,989 cekit        DEBUG    Retrieving module repositories for 'openshift-passwd'
2018-11-27 11:37:23,989 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,989 cekit        DEBUG    Adding module 'openshift-passwd', path: 'target/repo/cct_module.git-master/openshift-passwd'
2018-11-27 11:37:23,990 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/os-amq-jolokia/module.yaml'.
2018-11-27 11:37:23,993 cekit        DEBUG    Retrieving module repositories for 'os-amq-jolokia'
2018-11-27 11:37:23,993 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:23,993 cekit        DEBUG    Adding module 'os-amq-jolokia', path: 'target/repo/cct_module.git-master/os-amq-jolokia'
2018-11-27 11:37:23,994 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/os-amq-launch/module.yaml'.
2018-11-27 11:37:24,007 cekit        DEBUG    Retrieving module repositories for 'os-amq-launch'
2018-11-27 11:37:24,008 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:24,008 cekit        DEBUG    Adding module 'os-amq-launch', path: 'target/repo/cct_module.git-master/os-amq-launch'
2018-11-27 11:37:24,008 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/os-amq-optional/module.yaml'.
2018-11-27 11:37:24,031 cekit        DEBUG    Retrieving module repositories for 'os-amq-optional'
2018-11-27 11:37:24,031 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:24,031 cekit        DEBUG    Adding module 'os-amq-optional', path: 'target/repo/cct_module.git-master/os-amq-optional'
2018-11-27 11:37:24,032 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/os-amq-permissions/module.yaml'.
2018-11-27 11:37:24,035 cekit        DEBUG    Retrieving module repositories for 'os-amq-permissions'
2018-11-27 11:37:24,036 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:24,036 cekit        DEBUG    Adding module 'os-amq-permissions', path: 'target/repo/cct_module.git-master/os-amq-permissions'
2018-11-27 11:37:24,036 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/os-amq-s2i/module.yaml'.
2018-11-27 11:37:24,040 cekit        DEBUG    Retrieving module repositories for 'os-amq-s2i'
2018-11-27 11:37:24,041 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:24,041 cekit        DEBUG    Adding module 'os-amq-s2i', path: 'target/repo/cct_module.git-master/os-amq-s2i'
2018-11-27 11:37:24,042 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/os-amq-secure-mgmt-console/module.yaml'.
2018-11-27 11:37:24,047 cekit        DEBUG    Retrieving module repositories for 'os-amq-secure-mgmt-console'
2018-11-27 11:37:24,047 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:24,048 cekit        DEBUG    Adding module 'os-amq-secure-mgmt-console', path: 'target/repo/cct_module.git-master/os-amq-secure-mgmt-console'
2018-11-27 11:37:24,048 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/os-eap-activemq-rar/module.yaml'.
2018-11-27 11:37:24,069 cekit        DEBUG    Retrieving module repositories for 'os-eap-activemq-rar'
2018-11-27 11:37:24,071 cekit        DEBUG    No module repositories specified in descriptor
2018-11-27 11:37:24,071 cekit        DEBUG    Adding module 'os-eap-activemq-rar', path: 'target/repo/cct_module.git-master/os-eap-activemq-rar'
2018-11-27 11:37:24,071 cekit        DEBUG    Loading descriptor from path 'target/repo/cct_module.git-master/os-eap-db-drivers/module.yaml'.
2018-11-27 11:37:24,078 cekit        WARNING  The way of defining repository 'jboss-os' is deprecated. Convert it to an URL based repository object. Consult Cekit docs, for more details.
2018-11-27 11:37:24,079 cekit        ERROR    Package repository 'jboss-os' used in descriptor is not available in Cekit configuration file. 
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/cekit/cli.py", line 218, in run
    discover_modules(os.path.join(self.args.target, 'repo'))
  File "/usr/lib/python2.7/site-packages/cekit/module.py", line 92, in discover_modules
    'module.yaml'))))
  File "/usr/lib/python2.7/site-packages/cekit/descriptor/module.py", line 31, in __init__
    self._prepare()
  File "/usr/lib/python2.7/site-packages/cekit/descriptor/image.py", line 89, in _prepare
    self._descriptor['packages'] = Packages(self._descriptor['packages'])
  File "/usr/lib/python2.7/site-packages/cekit/descriptor/packages.py", line 48, in __init__
    self._prepare()
  File "/usr/lib/python2.7/site-packages/cekit/descriptor/packages.py", line 52, in _prepare
    for x in self._descriptor.get('repositories', [])]
  File "/usr/lib/python2.7/site-packages/cekit/descriptor/packages.py", line 66, in __init__
    descriptor = self._create_repo_object(descriptor)
  File "/usr/lib/python2.7/site-packages/cekit/descriptor/packages.py", line 105, in _create_repo_object
    descriptor['url']['repository'] = self._get_repo_url(descriptor)
  File "/usr/lib/python2.7/site-packages/cekit/descriptor/packages.py", line 128, in _get_repo_url
    % descriptor['name'])
CekitError: Package repository 'jboss-os' used in descriptor is not available in Cekit configuration file. 

```

**Comment by vorburger** [comment]
I'm also struggling to build it, but am hitting a different error, so separate issue, see #48.

**Comment by jmtd** [comment]
This has been fixed in cekit ≥ 2.2.4, please reopen if you continue having problems after upgrading your cekit.
